packages/fleet/lib/fleet.ts

```diff
@@ -53,7 +57,17 @@ export class Fleet {
     }
 
     public async rollout(commitId: CommitHash): Promise<Result<Deployment>> {
-        return deployments.create(this.dbClient.db, commitId);
+        return this.dbClient.db.transaction(async (trx) => {
+            const deployment = await deployments.create(trx, commitId);
+            if (deployment.isErr()) {
+                throw deployment.error;
+            }
+
+            // rolling out cancels all nodeConfigOverrides images
+            await nodeConfigOverrides.setForAll(trx, { image: this.nodeProvider.defaultNodeConfig.image });
```

Comment:
I think it is better not to have long running image override. We can use it to test on a single runner before rolling out all runners but the override would be short leaved and be removed next time the runners are rolled out.